artifacts/README.md

```diff
@@ -0,0 +1,51 @@
+# Layer static spec file
+[home](../README.md)
+* The artifact hold the entire layer spec and provided on layer creation as json body.
```

Comment:
The artifact hold -> Artifact holds layer spec and is provided on layer creation as json body.

Reply:
done